### ISSUE TITLE
After latest update HUE hub no longer found

### ISSUE THREAD

**Issue by robboz4** [ebaauw/homebridge-hue#721]
#### Issue
Updated Homebridge and Hue plugin. Homebridge did not run properly - UI X would not load accessories. Repretde message in log waiting 30 seconds before looking for a Hue Hub. I removed the Hue configuration and everything came up as normal.
homebridge-hue v0.11.72
homebridge-config-ui-x v4.23.1

node --version
v12.18.1

 Linux Homebridge 4.19.118-v7+ #1311

#### Log Messages

```
```

#### Debug Files

Jun 28 17:43:48 Homebridge homebridge[690]: [6/28/2020, 5:43:48 PM] [Hue] no bridges or gateways found - retrying in 30 seconds

Here is the relevant config section of the config.json file. This configuration has been stable for over 2 years.

{
Â Â Â Â Â Â Â Â Â Â Â Â "platform":Â "Hue",
Â Â Â Â Â Â Â Â Â Â Â Â "name":Â "Hue",
Â Â Â Â Â Â Â Â Â Â Â Â "users":Â {
Â Â Â Â Â Â Â Â Â Â Â Â Â Â Â Â "001788FFFE099838":Â "6tx3hL6K1dBE6CKcDTBlQRzBGpaISOXI73wUib01"
Â Â Â Â Â Â Â Â Â Â Â Â },
Â Â Â Â Â Â Â Â Â Â Â Â "sensors":Â false,
Â Â Â Â Â Â Â Â Â Â Â Â "excludeSensorTypes":Â [
Â Â Â Â Â Â Â Â Â Â Â Â Â Â Â Â "CLIPPresence",
Â Â Â Â Â Â Â Â Â Â Â Â Â Â Â Â "Geofence"
Â Â Â Â Â Â Â Â Â Â Â Â ],
Â Â Â Â Â Â Â Â Â Â Â Â "lights":Â true,
Â Â Â Â Â Â Â Â Â Â Â Â "philipsLights":Â true
Â Â Â Â Â Â Â Â },

It is a first generation Hue Hub.

The hub is working with the iPhone app.

**Comment by ebaauw** [comment]
By design Homebridge Hue delays the startup of Homebridge when it doesn’t find any bridges or gateways.  If it wouldn’t, HomeKit would delete all previously exposed accessories.

Not sure why your bridge isn’t found.  What’s the output of `ph discover -v`?

**Comment by robboz4** [comment]
pi@Homebridge:~ $ ph discover -v
{
  "deconz": {},
  "meethue": {},
  "upnp": {}
}
 Does this help?

**Comment by robboz4** [comment]
I got this from looking further at the ph command:

pi@Homebridge:~ $ ph -H xxx.xxx.xxx.xxx config
{
  "name": "Club-Robbo",
  "datastoreversion": "59",
  "swversion": "01043155",
  "apiversion": "1.16.0",
  "mac": "00:17:88:09:98:38",
  "bridgeid": "001788FFFE099838",
  "factorynew": false,
  "replacesbridgeid": null,
  "modelid": "BSB001"
}

It looks like the mac address is reflected  in the homebridge config correctly.

**Comment by ebaauw** [comment]
The bridge doesn’t seem to announce itself over UPnP, nor does it seem to register with the meethue portal.  Did you try and power cycle the bridge?

If that doesn’t help, the quick workaround is to specify the bridge’s IP address in config.json under `hosts`.  Not that the bridge needs to have a fixed IP address for this to work.

What else did you change besides updating Homebridge Hue?  Does Homebridge run on the same subnet as the Hue bridge?  Do you run a firewall that might be blocking the UPnP traffic and/or prevent the bridge from contacting the meethue portal?

**Comment by robboz4** [comment]
Hi!

Well firstly I updated the OS, then updated node etc... I needed to update node as the Hue plugin told me I  should upgrade.

I tried the power cycle option as that is always a goto quick fix.

But I notice that  the top LED is out on the hub. I think that is internet connectivity indicator so no idea why it would be out. Plus I can't fine Then I read the article that the V1 is no longer supported.

There has been no changes to my network set up in terms of firewalls etc. Everything is the same and was working up until the update. 
So looks like the Hue hub has gone!

With regards to the host  field, is that inside the Hue platform definition or is it a global homebridge entry? I've not come across that before.

Thanks.
